content/docs/use-cases/model-registry.md

```diff
@@ -0,0 +1,69 @@
+# Machine Learning Model Registry
+
+A **model registry** is a central catalog for data artifacts produced in
+multiple data science projects. Different models can be discovered, audited,
+shared, tested, and deployed from here. DVC works on top of Git to enable these
+capabilities on your existing ecosystem. And other tools from
+[Iterative](https://iterative.ai/) provide even better features and usability.
+
+![](/img/ml_model_registry_placement.jpg) _A common placement for model
+registries is between development and applications. See also [Data Registries]._
+
+[data registries]: /doc/use-cases/data-registries
+
+Model catalogs give your team key collaborative capabilities:
+
+- Store, organize, and track the [versions] of different models effectively.
+- Full data provenance and lineage trails guarantee that your modeling is
+  reproducible.
+- Share and control access to any data artifact, from early-stage [experiment]
+  results to production-ready models.
+- Save performance [metrics and plots] to quickly evaluate and compare models.
+- An interface to access specific model versions from other systems (e.g. for
+  CI/CD)
+
+[versions]: /doc/use-cases/versioning-data-and-model-files
+[experiment]: /doc/user-guide/experiment-management
+[metrics and plots]: /doc/start/metrics-parameters-plots
+
+Many of these benefits are built into DVC: Your [modeling process] and
+performance metadata become **codified** in Git-based <abbr>DVC projects</abbr>.
+This makes it possible to manage them with standard GitOps workflows along with
+code. Large model files are stored separately and can be pushed to [remote
+storage], an efficient and scalable access point.
+
+The relevant outputs of these model development projects can then be
+**packaged** together with their metadata, and published in a [global Git
+repository]. With DVC, existing model files stored remotely are reused, so
```

Comment:
We first mention "global Git repo" and link to the GTO section.

Reply:
GTO is not about global repos at all. I think we still make too much emphasis on the global repo setup. This is an advanced use case that would replace in CLU an upcoming Studio MR. I would not make the whole paragraph about this.

Reply:
Yes, the current text focuses on a "global" centralized registry that collects models from many projects, similar to the base case presented in https://dvc.org/doc/use-cases/data-registries but for models. I realize GTO is designed with source repos in mind though, so indeed this link text doesn't make sense.

But more broadly, I guess I should focus more on the cataloging aspect and less on the centralizing aspect in general.. ⌛

Reply:
* now that we're incorporating GTO